src/classes/yaml-utils.ts

```diff
@@ -59,8 +69,41 @@ export default class YamlUtils {
    * @param repoPath Repository path where .yaml files are located
    */
   public static printYaml = async (type: string, repoPath: string): Promise<void> => {
-    const file = YAML.parse(fs.readFileSync(`${repoPath + path.sep + type}.yaml`, "utf8"));
-    console.log(file);
+    try {
+      const file = YAML.parse(fs.readFileSync(`${repoPath + path.sep + type}.yaml`, "utf8"));
+      console.log(file);
+    } catch (err) {
+      Promise.reject(`Error when reading file: ${err}`);
+    }
+    
+  }
+
+  /**
+   * Attachs AWS into Minikube, if needed
```

Comment:
Does what now?

Reply:
Enables connectivity to an AWS database from Minikube deployment